src/Hl7.Fhir.Core.Tests/Rest/RequesterTests.cs

```diff
@@ -280,7 +279,7 @@ public async Tasks.Task TestBundleToEntryRequest()
             };
             bundleComponent.AddAnnotation(InteractionType.Search);
 
-            var entryRequest = await bundleComponent.ToEntryRequestAsync(_Settings);
+            var entryRequest = await bundleComponent.ToEntryRequestAsync(_Settings, ModelInfo.Version);
```

Comment:
What was version specific about `ToEntryRequestAsync` (I guess I will see later ;-))

Reply:
It is used for verifying the FhirVersion during communication.